activities/communication/communications-plan.md

```diff
@@ -0,0 +1,169 @@
+
+## Principles
+
+Our communications need to reflect our core values: open, quality, trust, community, helpful. They must also be non-partisan, in line with our [founding principles](../../organization/mission.md#founding-principles).
+
+Our messages range from neutral in tone to positive and inspirational. Our community won’t be sustainable if we attract people motivated by fear or anger.
+
+Our communications are in plain English, but we link to highly relevant pages in other languages, for example about our members’ work.
+
+Our communications should show to showcase the Foundation for Public Code as:
+
+* active, dynamic, getting things done
+* smart enough to be trusted with the importance and complexity of its work
+* plugged into and supportive of the network of visionaries and activists who also want this future
+* open and transparent, both about its intentions and operations as well as learnings, failures and successes
+
+We believe it’s more important to talk concretely about what we’ve already done than what we hope to do.
+
+## Objectives
+
+Our communications should help us achieve [our current priorities](../../organization/mission.md#current-priorities).
+
+## Audience
+
+Our audience is divided into:
+
+* current Foundation contributors
+* future Foundation contributors
+* influencers
+
+### Foundation contributors
+
+We believe [these are potential contributors to the Foundation for Public Code](https://about.publiccode.net/activities/value-and-impact/user-mapping/).
+
+### Influencers
+
+These are domain celebrities in open source, government digital transformation and civic tech.
+
+## Unique selling proposition
+
+We are the only organization focused on open source software for public organizations that:
+
+* is non-profit and member owned (unlike commercial open source govtech companies)
+* offers codebase stewardship for different projects (how are we different from the Commons Conservancy and the businesses they can put you in touch with?)
+* offers productization and marketing of the codebases (unlike Commons Conservancy)
+* actually provides guidance and support to help public orgs make their code open (unlike Free Software Foundation)
+* isn’t tied to the EU project lifecycle (unlike SCORE)
+* has a reproducible process suitable not just for cities, but all public organizations (unlike SCORE)
+* is international (unlike OS2, Cloud 4 Cities, VNG-Realisatie, Adullact)
+
+We are also actively building our ecosystem by:
+
+* connecting members with like-minded parties (potential members, partners, affiliates, structural funders)
+* boosting awareness of public code
+* setting standards and creating reusable resources (by providing thought leadership!)
+* demonstrating the value of public code with well documented, public business cases and risk analyses
+* nurturing our community online, and – where sensible – in person
+
+## Calls to action
```

Comment:
Not sure whether the content that follows are calls to action. When I see "call to action", I expect some kind of arguments that will make me want to do something e.g. Join us in the fight against climate change! 

When I read the content that follows below, I see different types of information that we communicate about to different groups/stakeholders. Could we come up with a heading that's more fitting?

Reply:
On a website, a call to action is [any action we invite the user to take](https://www.optimizely.com/nl/optimization-glossary/call-to-action/). A call to action shouldn't be persuasive; instead, it should help or prompt the user to accomplish a goal they have already formulated themselves (especially while they've read the preceding content).